src/mem/old_api.cc

```diff
@@ -439,6 +428,12 @@ Mem::Init(void)
      * that are never used or used only once; perhaps we should simply use
```

Comment:
```diff
-      * Starting with generic 2kB - 64kB buffr pools, then specific object types.
+      * Starting with generic buffer pools, then specific object types.
```

This PR makes comment a few lines above stale.

Reply:
Done in [0738130](https://github.com/squid-cache/squid/pull/1590/commits/07381308af2c283c803542334c4e6bdec64cb32c)